packages/lib/src/vis/utils.ts

```diff
@@ -177,19 +177,22 @@ export function getValueToIndexScale(
   );
 }
 
-export function getCanvasScale(
-  config: AxisConfig,
-  canvasSize: number
-): AxisScale {
+export function getDomainScale(config: AxisConfig, range: Domain): AxisScale {
```

Comment:
Added function to share the code between the hook used in tiling and `getCanvasSize`.
Any better idea for the name?

Reply:
`getAxisScaleFromConfig` ?

Reply:
Go for it